### PR TITLE
add examples for generic-schedule-params and schedule-status groupings

### PR DESCRIPTION
**Pull request by QiufangMa**

### CODE REVIEW

File: draft-ietf-netmod-schedule-yang.md [hunk 1]
```diff
@@ -449,6 +449,22 @@ This section uses the template described in {{Section 3.7 of ?I-D.ietf-netmod-rf
    just for the ease of understanding. Only the message body is provided with
    JSON used for encoding {{?RFC7951}}.
 
+## The "generic-schedule-params" Grouping
+
+   {{ex-0}} indicates the example of a requested schedule that needs to start no earlier than
+   08:00 AM, January 1, 2025 and end no later than 8:00 PM, January 31, 2025 (Beijing time).
+   Schedule requests fail to meet the requirements are ignored by the system.
```
Comment (boucadair) [inline draft-ietf-netmod-schedule-yang.md:456]:
```suggestion
   Schedule requests that fail to meet the requirements are ignored by the system.
```

Also, define what's a "system".

File: draft-ietf-netmod-schedule-yang.md [hunk 2]
```diff
@@ -642,9 +658,9 @@ This section uses the template described in {{Section 3.7 of ?I-D.ietf-netmod-rf
 ~~~~
 {: #ex-10 title="Example of Advanced iCalendar Recurrence"}
 
-  {{ex-11}} indicates a recurrence that occur every 20
-  minutes from 9:00 AM to 4:40 PM (UTC), with the occurrence starting at 10:20 AM
-  being excluded on 2025-12-01:
+   {{ex-11}} indicates a recurrence that occur every 20
```
Comment (boucadair) [inline draft-ietf-netmod-schedule-yang.md:661]:
```suggestion
   {{ex-11}} indicates a recurrence that occurs every 20
```

File: draft-ietf-netmod-schedule-yang.md [hunk 3]
```diff
@@ -660,6 +676,22 @@ This section uses the template described in {{Section 3.7 of ?I-D.ietf-netmod-rf
...
+## The "schedule-status" Grouping
+
+   {{ex-12}} indicates the scheduled recurrence status of {{ex-11}} at the time
+   of 12:15 PM, 2025-12-01 (UTC):
+
+~~~~
+{
+  "state": "enabled",
+  "version": 1,
+  "schedule-type": "recurrence",
```
Comment (boucadair) [inline draft-ietf-netmod-schedule-yang.md:688]:
```suggestion
  "schedule-type": "ietf-schedule:recurrence",
```

File: draft-ietf-netmod-schedule-yang.md [hunk 4]
```diff
@@ -660,6 +676,22 @@ This section uses the template described in {{Section 3.7 of ?I-D.ietf-netmod-rf
...
 {: #ex-11 title="Example of Advanced iCalendar Recurrence with Exceptions"}
 
+## The "schedule-status" Grouping
+
+   {{ex-12}} indicates the scheduled recurrence status of {{ex-11}} at the time
+   of 12:15 PM, 2025-12-01 (UTC):
+
+~~~~
+{
+  "state": "enabled",
```
Comment (boucadair) [inline draft-ietf-netmod-schedule-yang.md:686]:
```suggestion
  "state": "ietf-schedule:enabled",
```

**Review by boucadair — CHANGES_REQUESTED** [review]
Thanks.

File: draft-ietf-netmod-schedule-yang.md (hunk 1 above)
Comment (QiufangMa) [reply draft-ietf-netmod-schedule-yang.md:456]: > Also, define what's a "system".
I think it is a familiar comment, but I couldn't find where we also used the "system" and then defined it properly. My poor memory ;-) 
Would the following text be okay to add into the document where the word "system" is first used:
A system accepts and handles the schedule requests, which may help further automate the scheduling process of events, policy, services, or resources based on date and time.

Comment (boucadair) [reply draft-ietf-netmod-schedule-yang.md:456]: Looks good. Thanks.

**Pull request merged**